### ISSUE TITLE
Remove the pathway components as domain of ceterms:prerequisite

### ISSUE THREAD

**Issue by stuartasutton** [CredentialEngine/Schema-Development#813]
The [ceterms:prerequisite](https://purl.org/ctdl/terms/prerequisite) property still has a domain of the pathway component subclasses. Prerequisite in the context of pathways is a constraint. They need to be removed. I thought we were adding [ceterms:Course](https://purl.org/ctdl/terms/Course)

**Comment by siuc-nate** [comment]
We have an existing issue for this. See #807.

**Comment by stuartasutton** [comment]
OK, then can we act on it? I thought this was DONE.

**Comment by siuc-nate** [comment]
No, not yet. We also haven't sorted the problem (I believe it was mentioned in 807) of how to build a pathway where you want to say X precedes Y when the underlying data does not make that assertion.

**Comment by stuartasutton** [comment]
This can be closed. It is documented in https://github.com/CredentialEngine/Schema-Development/blob/master/PathwaysConditionsAndConstraints/proposal.txt
